tests/rptest/tests/cloud_retention_test.py

```diff
@@ -188,14 +189,15 @@ def check_total_size(include_below_start_offset):
             msg_size,
             readers=3,
             loop=True,
-            max_msgs=max_read_msgs,
+            max_throughput_mb=max_read_msgs,
```

Comment:
It still seems a little strange that we're using a number of messages as a rate in mb. Maybe this should be max_consume_rate_mb?

Reply:
yep, fixed that
no idea why it used like that before